workers/unity/Packages/com.improbable.gdk.querybasedinterest/Constraint.cs

```diff
@@ -0,0 +1,124 @@
+using System.Collections.Generic;
+using System.Linq;
+using Improbable.Gdk.Core;
+using UnityEngine;
+
+namespace Improbable.Gdk.QueryBasedInterest
+{
+    public static class Constraint
+    {
+        private static readonly List<ComponentInterest.QueryConstraint> EmptyList
+            = new List<ComponentInterest.QueryConstraint>();
+
+        private static ComponentInterest.QueryConstraint Default()
+        {
+            return new ComponentInterest.QueryConstraint
+            {
+                AndConstraint = EmptyList,
+                OrConstraint = EmptyList
+            };
+        }
+
+        public static ComponentInterest.QueryConstraint Sphere(double radius, Vector3 center)
+        {
+            var constraint = Default();
+            constraint.SphereConstraint = new ComponentInterest.SphereConstraint
+            {
+                Center = new Coordinates(center.x, center.y, center.z),
+                Radius = radius
+            };
+            return constraint;
+        }
+
+        public static ComponentInterest.QueryConstraint Cylinder(double radius, Vector3 center)
```

Comment:
yup it's infinitely tall - probably should ask runtime why it's done like that

Reply:
I think it's because we tend to think of most load balancing in 2d terms. We tend to not have that might going in that axis and the query is cheaper this way.